config/main.py

```diff
@@ -415,6 +422,11 @@ def add_vlan_member(ctx, vid, interface_name, untagged):
     if interface_name in members:
         if get_interface_mode() == "alias": 
             alias_name = interface_name_to_alias(interface_name)
+            if alias_name is None:
+                print "Invalid interface {}".format(interface_name)
+                sys.exit(0)
```

Comment:
Please replace `sys.exit(0)` `raise click.Abort()`, which will cause the program to exit with code 1.